### PR TITLE
[modules] Rename EAPI symbol to EMODAPI symbol

### PR DESCRIPTION
**Pull request by Coquinho**
Differentiate building a module and building a EFL library

### CODE REVIEW

File: src/modules/ecore_evas/engines/drm/ecore_evas_drm.c [hunk 1]
```diff
@@ -30,14 +30,14 @@
 
 #ifdef _WIN32
 # ifdef DLL_EXPORT
-#  define EAPI __declspec(dllexport)
+#  define EMODAPI __declspec(dllexport)
```
Comment (Coquinho) [inline src/modules/ecore_evas/engines/drm/ecore_evas_drm.c:33]: I don't know how modules work, is this correct?

File: src/modules/evas/engines/gl_common/evas_gl_core.h [hunk 2]
```diff
@@ -8,14 +8,10 @@
...
-#  ifdef DLL_EXPORT
-#   define EMODAPI __declspec(dllexport)
-#  else
-#   define EMODAPI
-#  endif
+# ifndef EFL_MODULE_STATIC
+#  define EMODAPI __declspec(dllexport)
 # else
-#  define EMODAPI __declspec(dllimport)
+#  define EMODAPI
```
Comment (JPTIZ) [inline src/modules/evas/engines/gl_common/evas_gl_core.h]: Wait: we're removing the `dllimport` possibility. Is this right (I didn't work too much with modules, so I sincerely don't know)?

Comment (Coquinho) [reply src/modules/evas/engines/gl_common/evas_gl_core.h]: @JPTIZ this was discussed in [slack](https://expertisesolutions.slack.com/archives/C7NGCQY8M/p1593524654020200?thread_ts=1593524654.020200).

Comment (JPTIZ) [reply src/modules/evas/engines/gl_common/evas_gl_core.h]: Thanks. Modules are never imported since they're loaded manually in runtime, so it makes sense.

**Pull request merged**